### ISSUE TITLE
Can't sign in to AMO

### ISSUE THREAD

**Issue by Tanath** [mozilla/addons#257]
### Describe the problem and steps to reproduce it:
Can no longer sign in to addons.mozilla.org. Clicked login at top.

### What happened?
Now when I click login it redirects me to firefox.com and then I get unknown account.

### What did you expect to happen?
For my old account to still work so I can manage my collection.

### Anything else we should know?
Javascript & cookies blocked by default but permitted for mozilla.org and firefox.com. I use a password manager so I know the login is correct.

**Comment by wagnerand** [comment]
You need to create a Firefox account first.

**Comment by Tanath** [comment]
You don't seem to understand the problem. I already have a mozilla account that I can no longer sign in to. I need access to it so I can manage my collections. I doubt creating a new account is going to give me access to it. If it would, that warrants opening an entirely new issue.

**Comment by wagnerand** [comment]
IF you get an 'unknown account' error on accounts.firefox.com, that means you don't have a Firefox Account. Note that a Firefox Account is independent from an AMO account.

**Comment by muffinresearch** [comment]
@Tanath the reason for this is the accounts on AMO have been replaced by Firefox Accounts and there was a period of overlap where users were prompted to migrate their accounts.

If you create a Firefox account with the email address of your old AMO account and login with that you should find things work as expected.

**Comment by Tanath** [comment]
My apologies, this did solve my issue. Thanks.
